.circleci/config.yml

```diff
@@ -13,13 +14,20 @@ jobs:
       - run:
           name: "gcloud auth"
           command: |
-            echo $ENCODED_GCR_KEY | base64 -d > $GOOGLE_APPLICATION_CREDENTIALS 
+            echo $ENCODED_GCR_KEY | base64 -d > $GOOGLE_APPLICATION_CREDENTIALS
+            echo "export GCLOUD_SERVICE_KEY=\$(echo \$ENCODED_GCR_KEY | base64 --decode)" >> $BASH_ENV
```

Comment:
you needed this to be able to do a docker push with the gcr?

Reply:
The first one was necessary to get google cloud logged in so that we could pull from a public bucket (it wasn't working for anonymous). The second one (GCLOUD_SERVICE_KEY) is necessary to pull intermediate images from our private GCR.